app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.errors import (NotWearingMaskError, VaccineError)
```

Comment:
The import statements should be organized according to the checklist: standard library imports first, followed by third-party imports, and then local application/library-specific imports. Ensure that the import order is correct.